simpleregex/func.py

```diff
@@ -6,29 +6,35 @@
 from simpleregex.models import RegEx
 
 
-def noneOrMany(what: RegEx):
+def none_or_many(what: RegEx):
     return ensure_regex(what) + "*"
 
 
-def oneOrMore(what: RegEx):
+def one_or_more(what: RegEx):
     return ensure_regex(what) + "+"
 
 
 def maybe(what: RegEx):
     return ensure_regex(what) + "?"
 
 
-def anyOfChar(items: str):
+def any_of_char(items: str):
     return RegEx("[" + items + "]")
 
 
+def regex_range(min: str, max: str):
+    return RegEx(["[", str(min), "-", str(max), "]"])
+
+
 def _wrap_regex(what: RegEx, prefix: str, suffix: str):
     what = ensure_regex(what)
-    what._patterns = [prefix, *what._patterns, suffix]
+    what._patterns = [*list(prefix), *what._patterns, *list(suffix)]
     return what
 
 
-def group(what: RegEx, name=None):
+def group(what: RegEx, name=None, non_capturing=False):
+    if non_capturing is True:
```

Comment:
why `is True` and not just `if x:`?

Reply:
To make this condition clear on what's expected. if x statement will be True if, x is anything that isn't False or None or 0.